challenge.md

```diff
@@ -31,20 +31,20 @@ Open up this file in your text editor, and write your answers below (where indic
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
- 
- <!-- Answer Ends Here -->
+ <!-- Answer Starts Here --> Relative
+  I am going from the current location of 'desktop'.
+ <!-- Answer Ends Here --> It is a relative path because it defines where I am going from the current location of 'desktop'.
```

Comment:
Sorry Wil, this is an Absolute path. This is because it defines the location (in this case, of 'Desktop') _independently of where we are_. Whether we're at `/`, `~`, or `/Users`, the path above will always refer to the same place.